_src/index.jade

```diff
@@ -9,9 +11,12 @@
         a.navbar__link(href="#section--reverse") Cachorro Rei
       li.navbar__item 
         a.navbar__link(href="#") Contato
+    <button class="openbtn" onclick="openNav()">&#9776; </button>
```

Comment:
Same here: 

button(class="openbtn" onclick="openNav()") `'` &#9776 `'`;

Reply:
I also fixed the class name to "navbar__openbtn" in this case (wasn't following BEM methodology like the rest)